js/categories.js

```diff
@@ -0,0 +1,52 @@
+const _catg = `
```

Comment:
What does `_catg` mean and why is it prefixed with an underscore?

Reply:
Not sure why I name stuff like that at times. Just a thing